test/helpers/RollbackTestHelper.sol

```diff
@@ -0,0 +1,212 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.30;
+
+import {SimpleContract} from "test/helpers/SimpleContract.sol";
+import {CompoundGovernorHelper} from "test/helpers/CompoundGovernorHelper.sol";
+import {IUpgradeRegressionManager} from "src/interfaces/IUpgradeRegressionManager.sol";
+
+/// @title RollbackTestHelper
+/// @notice Helper functions for testing rollback scenarios with SimpleContract
+contract RollbackTestHelper {
```

Comment:
I might name this `FakeProtocolRollbackTestHelper` or something similar (assuming you take my naming suggestion on what is currently called `SimpleContract`) to indicate that it is coupled to creating Rollback proposals related to that contract.